packages/http/src/HttpRemote.ts

```diff
@@ -1,23 +1,26 @@
 
 import { ErrorConverter, Request, Response as ResultResponse } from '@jitar/execution';
+import { AddWorkerRequest } from '@jitar/runtime';
 import type { Remote } from '@jitar/services';
 import { File } from '@jitar/sourcing';
+import { Validator } from '@jitar/validation';
 
 import HeaderKeys from './definitions/HeaderKeys';
 import HeaderValues from './definitions/HeaderValues';
-import { Validator } from '@jitar/validation';
 import InvalidWorkerId from './errors/InvalidWorkerId';
 
 export default class HttpRemote implements Remote
 {
     readonly #url: string;
+    readonly #fetch: (input: RequestInfo, init?: RequestInit) => Promise<Response>;
     
     readonly #errorConverter = new ErrorConverter();
     readonly #validator = new Validator();
 
-    constructor(url: string)
+    constructor(url: string, fetch: (input: RequestInfo, init?: RequestInit) => Promise<Response>)
```

Comment:
Is it an idea to create and an HttpClient interface with a default implementation that uses fetch? This way we can prevent mocking fetch, and is more consistent.